components/pipeline-service/production/stone-prd-m01/deploy.yaml

```diff
@@ -1712,6 +1712,27 @@ spec:
 ---
 apiVersion: external-secrets.io/v1beta1
 kind: ExternalSecret
+metadata:
+  annotations:
+    argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
+    argocd.argoproj.io/sync-wave: "-1"
+  name: tekton-chains-signing-secret
+  namespace: openshift-pipelines
+spec:
+  dataFrom:
+  - extract:
+      key: production/pipeline-service/stone-stage-m01/chains-signing-secret
```

Comment:
the key still says `stone-stage-m01` @Roming22

Reply:
Fixed